_includes/quick-guide-table.html

```diff
@@ -51,24 +51,22 @@
 
 
 <div class = "flex-container">
-    <div class="steps">1. Is SDA the right archive? <div class="arrow arrow-down"></div> </div> 
-    <div >Defining the information about the type of data that can be submitted</div>
-    <div class="steps">2. Create a submission request <div class="arrow arrow-down"></div> </div> 
-    <div > At this stage we are working with a limited number of datasets. If you want to be on the waiting list for submission, please apply <a href="https://nbis.se/support/supportform/index.php#sdaform">here</a> and we will contact you as soon as we are ready to accept new data submissions.</div>
-    <div class="steps">3. Get your submission account at central EGA <div class="arrow arrow-down"></div> </div>
-    <div>Fill the submission form and provide details of the data type(s), used platform(s) and estimated size of your submission. If you are affiliated to an existing consortium, such as the International Cancer Genome Project (ICGC), please add this information in the comments.</div>
-    <div class="steps">4. Register Study, DAC and Sample <div class="arrow arrow-down"></div> </div>
-    <div >Use the Submitter Portal to register your Study, Samples, Data Access Committee (DAC) and Policy</div>
-    <div class="steps">5. Encrypt and upload data <div class="arrow arrow-down"></div> </div>
-    <div >Encrypt your data files using the the instructions found <a href="https://github.com/NBISweden/EGA-SE-user-docs">here.</a> </div>
-    <div class="steps">6. Register experiments and runs <div class="arrow arrow-down"></div> </div>
-    <div >Associate each data file to a registered sample and study by Linking files to samples . Details of the experimental procedure you followed must be provided.</div>
-    <div class="steps">7. Finish your submission <div class="arrow arrow-down"></div> </div>
-    <div >Group your runs/analysis into datasets and link them to your new or existing DAC and policy using Submitter Portal 
-        or an XML based programmatic submission. Data request are done at a dataset level, thus files within a datasets must share release conditions.</div>
-    <div class="steps">8. Release </div>
-    <div>Instruct our Helpdesk to release your study. 
-        All registered studies are automatically placed on hold until the named submission or DAC contact instructs our Helpdesk for the study to be released</div>
+  <div class="steps">1. Is SDA the right archive? <div class="arrow arrow-down"></div></div> 
```

Comment:
I would prefer to have these lines split at 80 characters to make it more readable without the markdown parser. This is not important though, if you disagree! :)

Reply:
I agree